test/events/event_factory.js

```diff
@@ -1,112 +1,119 @@
+  const TOPIC_PARAMS = {
+    _oracle: accounts[1],
+    _name: ['Will Apple stock reach $300 by t', 'he end of 2017?'],
+    _resultNames: ['first', 'second', 'third'],
+    _bettingStartBlock: 40,
+    _bettingEndBlock: 60,
+    _resultSettingStartBlock: 70,
+    _resultSettingEndBlock: 90,
+  };
+  const NUM_OF_RESULTS = 4; // 3 results + invalid default
 
-    let addressManager;
-    let eventFactory;
-    let eventFactoryCreator = accounts[0];
-    let oracleFactory;
-    let topic;
-    let topicCreator = accounts[1];
+  let addressManager;
+  let eventFactory;
+  const eventFactoryCreator = accounts[0];
+  let oracleFactory;
+  let topic;
+  const topicCreator = accounts[1];
 
-    beforeEach(blockHeightManager.snapshot);
-    afterEach(blockHeightManager.revert);
+  beforeEach(blockHeightManager.snapshot);
+  afterEach(blockHeightManager.revert);
 
-    beforeEach(async function() {
-        addressManager = await AddressManager.deployed({ from: eventFactoryCreator });
-        
-        eventFactory = await EventFactory.deployed(addressManager.address, { from: eventFactoryCreator });
-        await addressManager.setEventFactoryAddress(eventFactory.address, { from: eventFactoryCreator });
-        assert.equal(await addressManager.getEventFactoryAddress(0), eventFactory.address);
+  beforeEach(async () => {
+    addressManager = await AddressManager.deployed({ from: eventFactoryCreator });
 
-        oracleFactory = await OracleFactory.deployed(addressManager.address, { from: eventFactoryCreator });
-        await addressManager.setOracleFactoryAddress(oracleFactory.address, { from: eventFactoryCreator });
-        assert.equal(await addressManager.getOracleFactoryAddress(0), oracleFactory.address);
-        
-        let transaction = await eventFactory.createTopic(...Object.values(TOPIC_PARAMS), { from: topicCreator });
-        topic = await TopicEvent.at(transaction.logs[0].args._topicAddress);
-    });
+    eventFactory = await EventFactory.deployed(addressManager.address, { from: eventFactoryCreator });
+    await addressManager.setEventFactoryAddress(eventFactory.address, { from: eventFactoryCreator });
+    assert.equal(await addressManager.getEventFactoryAddress(0), eventFactory.address);
+
+    oracleFactory = await OracleFactory.deployed(addressManager.address, { from: eventFactoryCreator });
+    await addressManager.setOracleFactoryAddress(oracleFactory.address, { from: eventFactoryCreator });
+    assert.equal(await addressManager.getOracleFactoryAddress(0), oracleFactory.address);
+
+    const transaction = await eventFactory.createTopic(...Object.values(TOPIC_PARAMS), { from: topicCreator });
+    topic = await TopicEvent.at(transaction.logs[0].args._topicAddress);
+  });
 
-    describe('constructor', async function() {
-        it('sets all the values', async function() {
-            assert.equal(await eventFactory.version.call(), 0);
-        });
+  describe('constructor', async () => {
```

Comment:
Like your changes in other PRs, no async keyword needed. There are a lot

Reply:
Oops, didn't mean to approve, but there is no major issues. Just reply to or address my comments.

Reply:
ya ill clean them up. was still learning how to use async/await properly before haha